components/datetime/tests/fixtures/tests/components-exact-matches.json

```diff
@@ -546,5 +546,23 @@
                 "en":  "week 52 of 2021"
             }
         }
+    },
+    {
+        "description": "Exact match for: MMMMW -> 'week' W 'of' MMMM",
```

Comment:
Question: Are there only exact matches possible here?

Reply:
Partial matches work too, same as week-of-year, I've added a test covering this. I did stumble upon a bug in get_best_available_format_pattern() as a result of this, I'll fix that in a separate PR.